### PR TITLE
Enhancement/project card

### PR DESCRIPTION
**Pull request by IgboPharaoh**
This pr adds:
- project icons to the project card component 
- Navigation links for the intended functionality of being able to navigate to the GitHub page and also to the good first issues page with the project title applied as a key.

### CODE REVIEW

**Comment by Extheoisah** [comment]
The alignment of the project cards is not consistent. Lets give them a fixed height so the github and good first issues component alignment is consistent across all cards.
image

File: components/page-level.tsx [hunk 1]
```diff
@@ -69,18 +66,50 @@ export default function PageLevel({
...
-                            <h3 className="text-4xl md:text-2xl font-semibold">
-                                {item.title}
-                            </h3>
-                            <p className="text-3xl md:text-xl font-semibold text-green">
-                                {item.subTitle}
-                            </p>
-                            <p className="flex flex-shrink-1 text-lg leading-[140%] md:text-justify">
-                                {item.description}
-                            </p>
+                        <div>
```
Comment (Extheoisah) [inline components/page-level.tsx:69]: does this `div` serve any purpose given we have the main `div` below?

**Comment by IgboPharaoh** [comment]
> The alignment of the project cards is not consistent. Lets give them a fixed height so the github and good first issues component alignment is consistent across all cards. image

I don't think we can give a fixed height because this component is used across multiple pages and the elements being rendered have varying heights most or which are higher than the project cards.

We can make the title, subTitle, and description section take up a `flex-1` which forces it to take up the full height and ensures the github and good first issues component are appropriately aligned. But in screens of `1106px` below it looks really big. Also to note the major cause of this displacement is the project title.

This is how it looks when make the title, subTitle, and description section take up a `flex-1` with screen size at a max-width of 1024px, the component are aligned but the size of some components becomes the issue.

Screenshot 2024-06-04 at 15 47 50

**Comment by Extheoisah** [comment]
> > The alignment of the project cards is not consistent. Lets give them a fixed height so the github and good first issues component alignment is consistent across all cards. image
> 
> I don't think we can give a fixed height because this component is used across multiple pages and the elements being rendered have varying heights most or which are higher than the project cards.
> 
> We can make the title, subTitle, and description section take up a `flex-1` which forces it to take up the full height and ensures the github and good first issues component are appropriately aligned. But in screens of `1106px` below it looks really big. Also to note the major cause of this displacement is the project title.
> 
> This is how it looks when make the title, subTitle, and description section take up a `flex-1` with screen size at a max-width of 1024px, the component are aligned but the size of some components becomes the issue.
> 
> Screenshot 2024-06-04 at 15 47 50

Hmm... tricky one eh! Let's move forward with what we have for now

**Review by Extheoisah — APPROVED** [review]

**Pull request merged**